### ISSUE TITLE
This extension should be closed.

### ISSUE THREAD

**Issue by ayoubkhan558** [formulahendry/vscode-auto-close-tag#160]
This extension should be closed because this functionality is built into VS Code by default.

**Comment by uglyeoin** [comment]
That's not why I installed it.  But I would love to know how I can auto close a tag by typing / if you know the answer please reveal

**Comment by ayoubkhan558** [comment]
> That's not why I installed it. But I would love to know how I can auto close a tag by typing / if you know the answer please reveal

You can use emmet, You can use it within any code editor, You have to install emmet plugin/extension for that particular code editor.. I sure you will love it. 
You don't how power and useful emmet is.
In VS code it is integrated by default. and for other code editors you can install it separately.

**Comment by sovelto-tommi** [comment]
> You don't how power and useful emmet is.
Sure you can use Emmet, **but** it would be nice to automatically change the matching tag just by typing in the new tag name - instead of having to invoke Emmet: Update tag command. Hence this extension, if it only would not suffer from such poor performance.

**Comment by ayoubkhan558** [comment]
> > You don't how power and useful emmet is.
> > Sure you can use Emmet, **but** it would be nice to automatically change the matching tag just by typing in the new tag name - instead of having to invoke Emmet: Update tag command. Hence this extension, if it only would not suffer from such poor performance.

Yes, I know about it, It`s also called Zen Coding. I already use it, that's why I said this extension is now used anymore.
